lib/chat_api_web/controllers/gmail_controller.ex

```diff
@@ -75,6 +75,7 @@ defmodule ChatApiWeb.GmailController do
       )
       |> case do
         {:ok, result} ->
+          notify_slack(conn)
```

Comment:
we can also pipe this with the line below :)
```elixir
conn |> notify_slack() |> json(%{ok: true, data: result})
```

Reply:
Done!